### PR TITLE
Add titles to groups of EVP tests

### PR DESCRIPTION
**Pull request by richsalz**
This builds on #3506 to add more titles and regroup some tests.

### CODE REVIEW

File: test/evpdigest.txt [hunk 1]
```diff
@@ -17,6 +17,8 @@
 # Which is currently unsupported by OpenSSL.  They were generated using the
 # reference implementation.  RFC7693 also mentions the 616263 / "abc" values.
 
+Title BLAKE tests
```
Comment (dot-asm) [inline test/evpdigest.txt]: Title =

Comment (richsalz) [inline test/evpdigest.txt]: oops.  updated commit pushed

**Comment by dot-asm** [comment]
It looks like there will be conflicts with #3506. How will you handle this? I'd suggest merge this and re-base #3506. Or even better bring all `exp*.txt` modifications here, harmonize, merge, then #3506 can be re-based and will becomes purely about code modifications...

**Comment by dot-asm** [comment]
Bottom line is that it would make the treating #3506 easier, please :-)

**Comment by richsalz** [comment]
okay, all Title changes are here and removed from #3506

**Comment by richsalz** [comment]
merged, thanks again.

**Pull request closed without merging**